### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NeOMakinG**
https://github.com/NeOMakinGOrg/cheapdeck-app/issues/1

### CODE REVIEW

**Comment by NeOMakinG** [comment]
test

**Review by matthieu-rolland — APPROVED** [review]
quality PR aproved

**Review by matthieu-rolland — COMMENTED** [review]
review comment

File: README.md [hunk 1]
```diff
@@ -22,3 +22,4 @@ npm run lint
 
 ### Customize configuration
 See [Configuration Reference](https://cli.vuejs.org/config/).
+test
```
Comment (matthieu-rolland) [inline README.md:25]: test feedback

**Pull request merged**